android/CouchbaseLite/src/androidTest/java/com/couchbase/lite/BlobTest.java

```diff
@@ -34,6 +38,9 @@
     final static String kBlobTestBlob1 = "i'm blob";
     final static String kBlobTestBlob2 = "i'm blob2";
 
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
```

Comment:
I'm wondering if we should move this to BaseTest.

Reply:
that will avoid declaring in many places. we can do that in a separate PR.